frontend/src/components/Contributions/ContributionListItem.vue

```diff
@@ -2,47 +2,100 @@
   <div class="contribution-list-item">
     <slot>
       <div class="border p-3 w-100 mb-1" :class="`border-${variant}`">
-        <div class="d-inline-flex">
-          <div class="mr-2"><b-icon :icon="icon" :variant="variant" class="h2"></b-icon></div>
-          <div v-if="firstName" class="mr-3">{{ firstName }} {{ lastName }}</div>
-          <div class="mr-2" :class="type != 'deleted' ? 'font-weight-bold' : ''">
-            {{ amount | GDD }}
+        <div>
+          <div class="d-inline-flex">
+            <div class="mr-2">
+              <b-icon
+                v-if="state === 'IN_PROGRESS'"
+                icon="question-square"
+                font-scale="2"
+                variant="warning"
+              ></b-icon>
+              <b-icon v-else :icon="icon" :variant="variant" class="h2"></b-icon>
+            </div>
+            <div v-if="firstName" class="mr-3">{{ firstName }} {{ lastName }}</div>
+            <div class="mr-2" :class="state !== 'DELETED' ? 'font-weight-bold' : ''">
+              {{ amount | GDD }}
+            </div>
+            {{ $t('math.minus') }}
+            <div class="mx-2">{{ $d(new Date(date), 'short') }}</div>
           </div>
-          {{ $t('math.minus') }}
-          <div class="mx-2">{{ $d(new Date(date), 'short') }}</div>
-        </div>
-        <div class="mr-2">
-          <span>{{ $t('contribution.date') }}</span>
-          <span>
-            {{ $d(new Date(contributionDate), 'monthAndYear') }}
-          </span>
-        </div>
-        <div class="mr-2">{{ memo }}</div>
-        <div v-if="type === 'pending' && !firstName" class="d-flex flex-row-reverse">
+          <div class="mr-2">
+            <span>{{ $t('contribution.date') }}</span>
+            <span>
+              {{ $d(new Date(contributionDate), 'monthAndYear') }}
+            </span>
+          </div>
+          <div class="mr-2">{{ memo }}</div>
           <div
-            class="pointer ml-5"
-            @click="
-              $emit('update-contribution-form', {
-                id: id,
-                contributionDate: contributionDate,
-                memo: memo,
-                amount: amount,
-              })
-            "
+            v-if="(state !== 'DELETED' && !firstName) || (state !== 'CONFIRMED' && !firstName)"
```

Comment:
```suggestion
            v-if="!['CONFIRMED', 'DELETED'].includes(state) && !firstName"
```

Alternative code. Hope this is correct …

I would not decide by the existenz of the "firstName" to show the edit, trash or message buttons!
Please make a property for the `ContributionList` and from there to the `ContributionListItem` which decides if the buttons should be shown …